<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicDownloader</class>
 <widget class="QDialog" name="MusicDownloader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MusicDownloader</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>681</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>歌曲API服务器：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="UrlIdLineEdit"/>
    </item>
    <item row="2" column="0" colspan="4">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QCheckBox" name="FrontArtistCheckBox">
        <property name="text">
         <string>歌曲名称前加歌手</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="RearArtistCheckBox">
        <property name="text">
         <string>歌曲名称后加歌手</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="LyricCheckBox">
        <property name="text">
         <string>歌曲是否下载歌词</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="HDCheckBox">
        <property name="text">
         <string>歌曲启用高清封面</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QComboBox" name="ModeComboBox">
      <property name="styleSheet">
       <string notr="true">QComboBox QAbstractItemView
{
	border: 2px solid darkgray;
	selection-background-color: lightgray;
}
QComboBox QAbstractItemView::item
{
	height:16px;
}</string>
      </property>
      <item>
       <property name="text">
        <string>网易云单曲</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>网易云歌单</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>QQ音乐单曲</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>QQ音乐歌单</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>网易云专辑</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>网易云歌手下载</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="2" colspan="2">
     <widget class="QLineEdit" name="APILineEdit"/>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="DownloadPushButton">
      <property name="text">
       <string>下 载</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="4">
     <widget class="QTextEdit" name="PrintTextEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(225, 225, 225);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
